### PR TITLE
feat(kubernetes): add option to specify kubeconfig file path

### LINKED ISSUE #2522: Specify kubeconfig file path for "trivy k8s" command
### Description

When you run the `trivy k8s` command, there is no option to specify an alternative `kubeconfig` file path. 
By default, the kubeconfig file resides in `$HOME/.kube/config`. 
However, users of Kubernetes often need to specify a different path, using a parameter like `--kubeconfig` on the `kubectl` command.

If you don't have the kubeconfig in the default filesystem path, then the error below is received.

```
root@trivy:~# trivy k8s --report summary cluster
2022-07-15T08:25:43.301Z        FATAL   get k8s artifacts error: Get "http://localhost:8080/api?timeout=32s": dial tcp [::1]:8080: connect: connection refused
```

### Solution

Please add `--kubeconfig` as an argument / parameter for the `trivy k8s` command.

### PR DESCRIPTION
**Pull request by mgsh**
## Related issues
- Close #2522

## Related PRs
- [ ] https://github.com/aquasecurity/trivy-kubernetes/pull/62

## Checklist
- [x] I've read the [guidelines for contributing](https://aquasecurity.github.io/trivy/latest/community/contribute/pr/) to this repository.
- [x] I've followed the [conventions](https://aquasecurity.github.io/trivy/latest/community/contribute/pr/#title) in the PR title.
- [ ] I've added tests that prove my fix is effective or that my feature works.
- [x] I've updated the [documentation](https://github.com/aquasecurity/trivy/blob/main/docs) with the relevant information (if needed).
- [x] I've added usage information (if the PR introduces new options)
- [ ] I've included a "before" and "after" example to the description (if the PR is a user interface change).

### CODE REVIEW

**Comment by josedonizetti** [comment]
@mgsh This looks good. Though let's wait until we merge the PR on trivy-kubernetes to update the dependencies and review this one. k?

File: go.mod
Comment (josedonizetti) [inline go.mod:326]: The PR was merged on trivy-kubernetes, can we fix the replace here?

**Comment by mgsh** [comment]
hi @josedonizetti, I've removed the mod replace change. Please review and merge when you get a chance.

**Comment by knqyf263** [comment]
@josedonizetti Can we merge it?

**Comment by josedonizetti** [comment]
@knqyf263 @mgsh Reviewing this now. Sorry was busy with some other things.

**Review by josedonizetti — APPROVED** [review]

**Comment by josedonizetti** [comment]
@knqyf263 can you review and merge?

**Review by knqyf263 — APPROVED** [review]

**Pull request merged**
